content/SCALE/SCALETutorials/Shares/SMB/SMBPrivateDatasetShare.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Setting Up SMB Private Dataset Shares"
+description: "Provides instructions on using SMB Private Datasets and Shares as an alternative to legacy SMB Home Shares."
+weight: 40
+aliases: 
+ - /scale/25.04/scaletutorials/shares/smb/addsmbhomeshare
+tags:
+- smb
+---
+
+{{< include file="/static/includes/RootLevelDatasetShareWarning.md" >}}
+
+{{< hint type=important title="Home Share Legacy Feature" >}}
+SMB Home Share is a legacy feature for organizations looking to maintain existing SMB configurations.
+TrueNAS removed the home share option from the SMB share **Puropse** list in 24.04 (Dragonfish).
+The SMB share **Other Options** includes a home share legacy option but it is not recommended for new shares, it is for organizations still using the legacy home shares option of adding a single SMB share to provide a personal directory for every user account.
+Future TrueNAS releases can introduce instability or require configuration changes affecting this legacy feature.
+
+Microsoft has eliminated the Home Share feature in Windows 11 and deprecated it in Windows 10.
+Microsoft support for Home Shares is planned for October 2025.
```

Comment:
This seems a bit unclear, as it makes it seems like Microsoft adds support for Home Shares in a couple of months.

Reply:
Good catch!